Assets/_Application/Layers/LayerTypes/GeoJsonLayers/GeoJsonLayerGameObject.cs

```diff
@@ -186,15 +187,16 @@ private void CreateFeatureMappings(IGeoJsonVisualisationLayer layer, Feature fea
                     {
                         subObject.transform.position = verts[0];
                         GeoJSONPointLayer pointLayer = layer as GeoJSONPointLayer;
-                        subObject.AddComponent<SphereCollider>().radius = pointLayer.PointRenderer3D.MeshScale * 0.5f;
+                        //subObject.AddComponent<SphereCollider>().radius = pointLayer.PointRenderer3D.MeshScale * 0.5f;
 
                     }
                 }
 
-                               
                 mesh.RecalculateBounds();
                 meshes[i] = mesh;
 
+                Vector3 unityPosition = subObject.transform.position;
+                Coordinate coord = new Coordinate(CoordinateSystem.Unity, unityPosition.x, unityPosition.y, unityPosition.z); //before parenting so worldposition
```

Comment:
volgens mij moest je voor een Unity Coordinate system een andere constructor gebruiken (niet logisch weet ik) ik zou even met Martijn bespreken of dit inmiddels zo kan om zeker te weten dat dit goed gaat, ik heb hier ooit eens een hele vervelende bug mee gehad